oss.krtirtho.Spotube.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>oss.krtirtho.spotube</id>
+  <name>Spotube</name>
+  <summary>
+  A lightweight free Spotify desktop-client which handles playback manually, streams music using Youtube &amp; no Spotify premium account is needed
+  </summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD 4-Clause</project_license>
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+  <developer_name>Kingkor Roy Tirtho</developer_name>
+  <url type="homepage">https://github.com/krtirtho/spotube</url>
+  <description>
+    <p>Spotube is a Flutter based lightweight spotify client. It utilizes the power of Spotify &amp; Youtube's public API &amp; creates a hazardless, performant &amp; resource friendly User Experience</p>
```

Comment:
```suggestion
    <p>Spotube is a lightweight Spotify client. It utilizes the power of Spotify &amp; Youtube's public API &amp; creates a hazardless, performant &amp; resource friendly User Experience</p>
```

user don't care about the framework this is written it.

Reply:
It's one of the key feature of this application. Similar apps mostly uses PyGtk/PyQt/Electron frameworks & Flutter makes this application unique. And I think I should let my users know the biggest feature of the application😃